spec/models/service/retirement_management_spec.rb

```diff
@@ -1,19 +1,34 @@
 describe "Service Retirement Management" do
-  let(:user) { FactoryGirl.create(:user_miq_request_approver, :userid => "admin") }
+  let!(:user) { FactoryGirl.create(:user_miq_request_approver, :userid => "admin") }
```

Comment:
thanks. This is better than the cop-out and saying this user can do `"everything"`